providers/class.two-factor-email.php

```diff
@@ -51,7 +51,7 @@ function authentication_page( $user ) {
 		<p><?php esc_html_e( 'A verification code has been sent to the email address associated with your account.', 'two-factor' ); ?></p>
 		<p>
 			<label for="authcode"><?php esc_html_e( 'Verification Code:' ); ?></label>
-			<input type="tel" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
+			<input type="text" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
```

Comment:
Leaving this as `tel` results in a numpad input on mobile.  If you're going to change the input type, please change to an input type that keeps this ux.